assets/helpers/paymentIntegrations/paymentApi.js

```diff
@@ -9,11 +9,12 @@ import {
   type AcquisitionABTest,
```

Comment:
Sorry to be a pain, but although the comments are now accurate this file name is still misleading

Reply:
What do you suggest?

Reply:
readerRevenueApis / contributionsApis?